python/setup.py

```diff
@@ -10,12 +11,12 @@
         "../src/dynamic_model.cc",
         "../src/utility.cc"
     ],
-    include_dirs = ['/usr/local/include/eigen3', '../include', 'cpp/'],
+    include_dirs = ['/usr/local/include/eigen3', '../include', 'cpp/', numpy.get_include()],
```

Comment:
(an issue from before, but...) Why are we hardcoding the path to eigen3 here. IIRC it is header only, so do we need the user to install it?

Reply:
Merge for now, checking this added as a todo